keps/sig-release/1731-publishing-packages/README.md

```diff
@@ -135,46 +149,47 @@ Scenario: [...]
 
 ### Implementation Details/Notes/Constraints
 
-Packages will be published for different ...
-- **`${dist}`**: a combination of `<distribution>-<code-name-or-version>`
-  (e.g. `debian-jessie`, `ubuntu-xenial`, `fedora-23`, `centos-7`, ...)
+Packages will be published for different:
+
+- Managers, like `apt` (consumes deb packages) and `yum` (consumes rpm packages).
 - **`${k8s_release}`**: the version of kubernetes `<major>.<minor>`
   (e.g. `1.12`, `1.13`, `1.14`, ...)
 - **`${channel}`**: can be `stable`, `dev`, `nightly`
-    - `stable`: all official releases for `${k8s_release}`
-      (e.g.: `1.13.0`, `1.13.1`, `1.13.2`, ...)
-    - `dev`: all development releases for all minor releases in this `${k8s_release}`, including `alpha`s, `beta`s and `rc`s
-      (e.g.: `1.13.0-rc.2`, `1.13.2-beta.0`, `1.13.1-alpha.3`, ...)
-    - `nightly`: any package cut automatically on a daily basis
-
+  - `stable`: all official releases for `${k8s_release}`
+    (e.g.: `1.13.0`, `1.13.1`, `1.13.2`, ...)
+  - `dev`: all development releases for all minor releases in this `${k8s_release}`, including `alpha`s, `beta`s and `rc`s
+    (e.g.: `1.13.0-rc.2`, `1.13.2-beta.0`, `1.13.1-alpha.3`, ...)
+  - `nightly`: any package cut automatically on a daily basis (optionally)
 
-This means, that end-users can configure their systems’ package managers to use those different `${channel}`s of a kubernetes `${k8s_release}` for their `${dist}`.
+This means, that End Users can configure their systems’ package managers to use
+those different `${channel}`s of a kubernetes `${k8s_release}` for their
+corresponding package manager.
 
 A configuration for the package managers might look something like:
 
 - deb:
-    ```
-    # deb http://apt.kubernetes.io/${dist} ${k8s_release} ${channel}
-    deb http://apt.kubernetes.io/debian-jessie 1.13 nightly
-    ```
+  ```
+  # deb http://apt.kubernetes.io ${k8s_release} ${channel}
+  deb http://apt.kubernetes.io/debian-jessie 1.13 nightly
```

Comment:
Aside: I'd expect to see something like
```
deb [signed-by=/etc/keyrings/kubernetes-keyring.gpg] http://apt.kubernetes.io/debian-jessie 1.26 nightly
```

Reply:
Good, I'll add this example by modifying the existing one.